lang/python/python3/Makefile

```diff
@@ -284,16 +284,6 @@ endef
 Py3Package/python3-light/install:=:
 Package/python3/install:=:
 
-# libuuid is provided by e2fsprogs and uuid/uuid.h is moved into
-# $(STAGING_DIR_HOST)/include/e2fsprogs
-HOST_CFLAGS += \
```

Comment:
these CFLAGS/CPPFLAGS looks more recent  44fb4927f11add18baa11617e67c8a697a3f528d  `python3: Fix uuid module not compiled for host Python`

i have no idea about this; or i forgot;

Reply:
Yeah libuuid is no longer provided by e2fsprogs.

Reply:
If you check the CI build logs, you will see that the _uuid module for host Python is not built, with this error:

```
/builder/build_dir/hostpkg/Python-3.11.7/Modules/_uuidmodule.c:11:12: fatal error: uuid.h: No such file or directory
   11 |   #include <uuid.h>
      |            ^~~~~~~~
compilation terminated.
```

I agree that adding `$(STAGING_DIR_HOST)/include/e2fsprogs` to the include dirs list isn't useful anymore, but something else needs to be done to fix this build error. (I don't have time right now to look into this further.)

Reply:
It should be using uuid/uuid.h and not just uuid.h

actually I just checked the file. It should be matching to the second part of the if.

Reply:
I found the issue. That if should be reversed IMO.

So the OS has /usr/include/uuid.h, which the configure script checks and tries to use. It should be reversed honestly.

Anyway

```
   1 /usr/bin/ld: /home/mangix/devstuff/openwrt/staging_dir/host/lib/libuuid.a(la-gen_uuid.o): relocation R_X86_64_TPOFF32 against `state_fd.13' can not be us     ed when making a shared object; recompile with -fPIC
1445 /usr/bin/ld: failed to set dynamic section sizes: bad value
```

Instead of using meson, the util-linux tools package uses autoconf, which builds PIC shared libraries but not static. Lovely.